go-controller/pkg/ovn/pods_test.go

```diff
@@ -1307,7 +1307,7 @@ var _ = ginkgo.Describe("OVN Pod Operations", func() {
 								// check old value for iface-id-ver will be updated to pod.UID
 								"iface-id-ver": "wrong_value",
 							},
-							PortSecurity: []string{t1.podMAC, t1.podIP},
+							PortSecurity: []string{fmt.Sprintf("%s %s", t1.podMAC, t1.podIP)},
```

Comment:
yup, how did this ever work?!? :P

Reply:
It's because in 4.11 current code we would update all non default fields, which meant even though the test case started with the wrong inital db, our addLogicalPort would update the port security field so it would work. When I changed the code to only update explicit fields I left out port security, so then because that field wasnt getting updated, the test case was failing :) (note I fixed it so we do update port security field now)